packages/sitecore-jss/src/site/site-resolver.ts

```diff
@@ -3,10 +3,25 @@ import { SiteInfo } from './graphql-siteinfo-service';
 // Delimiters for multi-value hostnames
 const DELIMITERS = /\||,|;/g;
 
+export type SiteResolverType = {
```

Comment:
Why do we need a new type? Can't we just use `SiteResolver` in the same places you are using this?